docs/am43xx.md

```diff
@@ -0,0 +1,57 @@
+# OP-TEE on Texas Instruments AM43xx
+
+# Contents
+1. [Introduction](#1-introduction)
+2. [Regular build](#2-regular-build)
+3. [Booting the device](#3-booting-the-device)
+
+# 1. Introduction
+The instructions here will tell how to run OP-TEE on the AM43xx-EVM and
+the EPOS Reference Design board.
+Secure TI devices require a boot image that is authenticated by ROM
+code to function. Without this, even JTAG remains locked. In order to create
+a valid boot image for a secure device from TI, the initial public software
+image must be signed and combined with various headers, certificates, and
+other binary images.
+
+Information on the details on the complete boot image format can be obtained
+from Texas Instruments. The tools used to generate boot images for secure
+devices are part of a secure development package (SECDEV) that can be
+downloaded from:
+
+	http://www.ti.com/mysecuresoftware (login required)
```

Comment:
I've never used it myself, this is just what the marketing folks tell me to use (as TI employees we can acquire the SECDEV through other means).